tests/cli/commands/test_dag_command.py

```diff
@@ -139,6 +140,19 @@ def test_show_dag_print(self):
         self.assertIn("graph [label=example_bash_operator labelloc=t rankdir=LR]", out)
         self.assertIn("runme_2 -> run_after_loop", out)
 
+    def test_generate_dag_yaml(self):
+
+        directory = "/tmp/airflow_dry_run_test/"
```

Comment:
Can you use [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) to avoid side-effect?